resources/views/layouts/components/head.blade.php

```diff
@@ -3,7 +3,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{{ (!empty($pageTitle) ? $pageTitle . ' · ' : '') . config('app.name') }}</title>
-    <link rel="icon" type="image/png" href="{{ asset('assets/images/favicon.webp') }}">
+    <link rel="icon" type="image/png" href="{{ asset((app()->environment('local') || app()->environment('stage')) ? 'assets/images/favicon_dev.webp' : 'assets/images/favicon.webp') }}">
```

Comment:
I used "stage" because there is a reference to this environment here:

https://github.com/RetroAchievements/RAWeb/blob/master/app/Site/Middleware/ForceHttps.php#L15

Should this be changed as well?

Reply:
oh my, you're right, it's actually configured with `stage` as environment 🙈 
checked on the server, sorry, i remembered that incorrectly.

Reply:
No worries, set back to `stage` in https://github.com/RetroAchievements/RAWeb/pull/1519/commits/02bc8dd91e7541f731d109dad302ed6fef10f987